File: Beginner/1010 (Simple Calculate)/1010.java
import java.io.IOException;
import java.util.Scanner;
import java.text.DecimalFormat;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);
        DecimalFormat df = new DecimalFormat("0.00");

        int a = ler.nextInt();
        int b = ler.nextInt();
        double e = ler.nextDouble();
        int c = ler.nextInt();
        int d = ler.nextInt();
        double f = ler.nextDouble();
        
        
        double result = (e*b + d * f);
        System.out.println("VALOR A PAGAR: R$ " + df.format(result));
        
    }
 
}

File: Beginner/1010 (Simple Calculate)/1010.cpp
#include <iostream>
#include <stdio.h>
using namespace std;
int main()
{
    int A,B,C,D;
    double E,F;
    cin >> A >> B >> E >> C >> D >> F;
    printf("VALOR A PAGAR: R$ %.2f\n",E*B+D*F);
    system("pause");
    return 0;
}
